src/axmysql/messagestore/store_test.go

```diff
@@ -0,0 +1,55 @@
+package messagestore_test
+
+import (
+	"database/sql"
+	"os"
+
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/jmalloc/ax/src/ax/messagestore"
+	"github.com/jmalloc/ax/src/ax/persistence"
+	"github.com/jmalloc/ax/src/axmysql"
+	"github.com/jmalloc/ax/src/axmysql/internal/schema"
+	mysqlmessagestore "github.com/jmalloc/ax/src/axmysql/messagestore"
+	"github.com/jmalloc/ax/src/axtest/messagestoretests"
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = Describe("MessageStore", func() {
```

Comment:
This name should refer to the subject under test -- so just `Store` in this case.